### ISSUE TITLE
2020-04-RC: Stop Sending Reports to the DEBUG File

### ISSUE THREAD

**Issue by OPMUSER** [OPM/opm-simulators#2597]
No standard reports should be sent to the debug file, as it clutters up the file. Also the "echoing" of the input read should also **not** be sent to the debug file. The debug file should only be used for debug output.

At the moment the debug and print files are very similar.

**Comment by atgeirr** [comment]
The principle has been that the debug file gets **everything** so that nothing is missing. I think that is a good principle, but also agree that the reports for all regions can be a bit annoying. There are also other printouts going to the debug file that can be overwhelming (MSW and VFP, I am looking at you...).

I suggest that we follow this rule of thumb: User friendliness is the top priority for the stdout and PRT logs, developer friendliness is the top priority for the DBG log.

As an aside I miss having a log file that contains exactly the same as the stdout output, since that is the most compact log (no well reports etc.) I have suggested to store this as a .LOG file, but apparently that was not good (don't remember why). What would be a good way to store this? Another possibility is to add the ability to turn off certain logging features for a run, by adding a command line option.

**Comment by OPMUSER** [comment]
> I suggest that we follow this rule of thumb: User friendliness is the top priority for the stdout and PRT logs, developer friendliness is the top priority for the DBG log.

Agreed, the DEBUG file is for the developers, so there is no need to have the standard reports going there.

> As an aside I miss having a log file that contains exactly the same as the stdout output, since that is the most compact log (no well reports etc.) I have suggested to store this as a .LOG file, but apparently that was not good (don't remember why). What would be a good way to store this? Another possibility is to add the ability to turn off certain logging features for a run, by adding a command line option.

OPMRUN does this automatically, by `| tee` the terminal output to *.LOG., but you can always roll your own.

Also , please, please no more command line parameters we already have 138. My target for 2020 was to get this below 100, but so far I have failed miserably :-)

[MOD02-WSEG01-OPM2004-R01.LOG](https://github.com/OPM/opm-simulators/files/4592129/MOD02-WSEG01-OPM2004-R01.LOG)

**Comment by OPMUSER** [comment]
I still would like to see the the printed reports deleted from the DEBUG file and the timing report moved into the DEBUG file instead of creating more files. The augment for a separate file would go away once we get rid of the reports from the DEBUG file.
